### ISSUE TITLE
Losing Exe Memory Access 

### ISSUE THREAD

**Issue by HeavenHeal** [ReClassNET/ReClass.NET#68]
i'm trying to attach to Reclass.Net to a game Called *Black Desert* i can successfully attach to the game right away after running the process but in 10 secs i lose the ability to scan the memory / Check the modules with 0 Error Message.. i was able to attach the Normal Reclass without any problem as you can see in the attached image that ReClass.NET still attached to the process but i can't view it's modules or run memory scan also tried with the pipe plugin but it's crashing the exe after attaching ( tried attaching to other exe files but crashing also ) and wasn't able to use Handle Hijacking plugin can you tell me exactly how to use it ? 

![image](https://user-images.githubusercontent.com/29453204/40881346-69338622-66c4-11e8-9ea7-75f5895cb3f5.png)

**Comment by KN4CK3R** [comment]
Does the game use an anti cheat system which removes handles to the process? Maybe not if the other ReClass doesn't have problems.

**Comment by HeavenHeal** [comment]
yes it uses XignCode3 but that doesn't remove the handle from the process as ArtMoney, CrySearch, CE and the normal Reclass working without any problem

**Comment by KN4CK3R** [comment]
Can I use the 7 day trial to test it or do I need the full game?

**Comment by HeavenHeal** [comment]
yes u can use the 7 day trial

**Comment by KN4CK3R** [comment]
Ok, I will test it later :zzz:

**Comment by HeavenHeal** [comment]
Sure Thanks ^^

**Comment by Timboy67678** [comment]
Xigncode3 strips handles, so I'm pretty sure that's why you arent getting
access to memory

**Comment by KN4CK3R** [comment]
But then the other tools shouldn't work too.

**Comment by Timboy67678** [comment]
Is that tested by you?

**Comment by KN4CK3R** [comment]
![grafik](https://user-images.githubusercontent.com/1666336/40885502-b2fe3046-6727-11e8-8de0-c4e6e7af698f.png)
I don't have problems. After which time do you loose access?

**Comment by HeavenHeal** [comment]
after xigncode completely loading weird that you have it working after loading ummm i also tried compiling from the source and used the version from the releases

**Comment by Timboy67678** [comment]
i dont think xigncode stripes handle access from all things, only those it deems as "Hack tool". i could be wrong, could be a fluke, could be any number of things.

**Comment by KN4CK3R** [comment]
When does it stop working for you?

![grafik](https://user-images.githubusercontent.com/1666336/40938024-d57f39e8-6840-11e8-93ca-d76c09f259d8.png)
Here I'm connected to a server and other players are visible. So I think I'm "ingame" and xign should have done it's work. I don't know why it isn't working for you. I think you run ReClass as admin? (otherwise the process shouldn't be visible because it runs as admin too.)

**Comment by HeavenHeal** [comment]
https://www.youtube.com/watch?v=Eve7HYZ8B3Q&feature=youtu.be 

here is a video for what's actually going on... btw i'm on 
![image](https://user-images.githubusercontent.com/29453204/40942928-d50e166e-684f-11e8-99f7-9991b67b12dd.png)

gonna setup a vmware with other Windows version and test

**Comment by KN4CK3R** [comment]
Maybe Xign doesn't get loaded for me / the 7 day trial. I don't remember the Xign window while starting the game.

**Comment by HeavenHeal** [comment]
i'm testing with the 7 days trial

**Comment by KN4CK3R** [comment]
Try to wait until you are ingame and attach ReClass then.

**Comment by HeavenHeal** [comment]
same result mate and will give an error ( failed to get process modules )

**Comment by HeavenHeal** [comment]
same result mate and will give an error ( failed to get process modules )
